src/python/pants/scm/subsystems/changed.py

```diff
@@ -156,18 +144,23 @@ def register_options(cls, register):
             type=DependeesOption,
             default=DependeesOption.NONE,
             help="Include direct or transitive dependees of changed targets.",
-            removal_version="2.0.1.dev0",
+            removal_version="2.1.0.dev0",
             removal_hint="Use `--changed-dependees` instead of `--changed-include-dependees`.",
         )
         register(
             "--fast",
             type=bool,
             default=False,
             help="Stop searching for owners once a source is mapped to at least one owning target.",
-            removal_version="2.0.1.dev0",
+            removal_version="2.1.0.dev0",
             removal_hint="The option `--changed-fast` no longer does anything.",
         )
 
 
 def rules():
-    return [find_owners, RootRule(ChangedRequest)]
+    return [
+        map_addresses_to_dependees,
+        find_dependees,
```

Comment:
So this is a little weird. We need the `dependees.py` rules to be always in the build graph because they are now always used for resolving addresses. Without this, most unit tests fail due to an incomplete rule graph.

I wasn't sure where to register the rules, among these 3 options:
1) Move the dependees logic out of `backend/project_info/dependees.py`, e.g. into `graph.py`. Always activate from there.
2) Export the rules from `changed.py`, because we already always register these rules.
3) Directly import these two rules from `engine_initializer.py`.

I didn't go with #1 because I couldn't think of a good place for those two rules. It makes sense to me to have the logic for dependees in `dependees.py`. So, I went with #2.

Reply:
Seems fine to me, and we can always change this later. Nice thing about the engine is that you don't import rules in order to use them, so moving rule code around doesn't require deprecation!